fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformServiceImpl.java

```diff
@@ -1428,15 +1428,15 @@ public LoanAccountData retrieveLoanProductDetailsTemplate(final Long productId,
             }
         }
 
-        Collection<LoanAccountSummaryData> clientActiveLoanOptions = null;
-        if(loanProduct.canUseForTopup() && clientId != null){
-            clientActiveLoanOptions = this.accountDetailsReadPlatformService.retrieveClientActiveLoanAccountSummary(clientId);
```

Comment:
We shouldn't be removing the existing condition which checks for client loans, rather we should be adding an else if for additionally checking for group loans